src/main/java/org/snomed/snowstorm/core/data/services/TraceabilityLogService.java

```diff
@@ -159,18 +159,27 @@ void logActivity(String userId, final Commit commit, final PersistedComponents b
 			}
 		}
 		for (Description description : persistedDescriptions) {
+			final long conceptId = parseLong(description.getConceptId());
```

Comment:
I've added several null checks to TraceabilityLogService. Whenever I imported a DELTA file, the class threw NPEs. @kaicode has mentioned that supporting concrete Relationships in Traceability is a part of Phase 3.